includes/wpmenucart-settings.php

```diff
@@ -563,20 +563,20 @@ public function select_element_callback( $args ) {
 
 		$disabled = (isset( $args['disabled'] )) ? ' disabled' : '';
 		
-		$html = sprintf( '<select name="%1$s[%2$s]" id="%1$s[%2$s]"%3$s>', $menu, $id, $disabled );
-		$html .= sprintf( '<option value="%s"%s>%s</option>', '0', selected( $current, '0', false ), '' );
+		$html  = sprintf( '<select name="%1$s[%2$s]" id="%1$s[%2$s]"%3$s>', esc_attr( $menu ), esc_attr( $id ), esc_attr( $disabled ) );
+		$html .= sprintf( '<option value="%s"%s>%s</option>', '0', selected( esc_attr( $current ), '0', false ), '' );
```

Comment:
@alexmigf I think `selected()` can only output `selected="selected"`, so escaping the input parameters for it does nothing.